src/github/index.js

```diff
@@ -184,17 +165,17 @@ export default class GitHub {
   }
 
   async closeIssueIfExists({ title, comment, labels }) {
-    const existingIssues = await this.searchIssues({ ...this.commonParams, title, q: `is:issue is:${ISSUE_STATE_OPEN} label:${labels.join(',')}` });
+    const openedIssues = await this.searchIssues({ ...this.commonParams, title, labels, state: ISSUE_STATE_OPEN });
 
-    if (existingIssues) {
-      for (const existingIssue of existingIssues) {
-        await this.octokit.rest.issues.update({ ...this.commonParams, issue_number: existingIssue.number, state: ISSUE_STATE_CLOSED }); // eslint-disable-line no-await-in-loop
-        await this.addCommentToIssue({ ...this.commonParams, issue_number: existingIssue.number, body: comment }); // eslint-disable-line no-await-in-loop
-        logger.info(`🤖 Github issue closed for ${title}: ${existingIssue.html_url}`);
+    if (openedIssues) {
```

Comment:
I'm in favor of using early exits every time we can, it's more idiomatic and I find it more readable.

Reply:
done